scripts/notify-lark.js

```
@@ -0,0 +1,117 @@
+
+    return {
+        timeSec,
+        Signature,
+    };
+}
+
+const chatURL = process.env.RABBY_LARK_CHAT_URL;
+const secret = process.env.RABBY_LARK_CHAT_SECRET;
+
+if (!chatURL) {
+    throw new Error('LARK_CHAT_URL is not set');
```

ESLint

[error] Replace `····` with `··`

(prettier/prettier)